script.js

```diff
@@ -0,0 +1,231 @@
+
+class Player {
```

Comment:
Our player lacks some functionality and feature. For one to make our `Board` methods simpler our player needs to have a storage of all its made moves. Another is the player object should have a method that returns the index a user chooses.

Reply:
Hi Ryan,
I want to tell you that my partner and I did almost the suggestions that you made in your last review only the suggestion about the player wasn't clear  for us, you ask to store the player moves in the player object, in this case i will have an arrays for each player it means that i will have a lot of empty spots, can you be more explicit?
thanks.

Reply:
I see you converted your Player into a class. That's not what I meant.

Ok consider this suggested factory for Player:

```javascript
const Player = (name, symbol) => {
  const moves = []; // this is an array where we store the indices made by the player
                     // during the course of the game
 
// the following function is what happens whenever the player makes a move
 const turn = (index, board) => {
    moves.push(index);
    board.markSquare(index, symbol);
  }

 return {
   name,
  symbol,
  moves,
  turn
 }
}
```